students/allan-lee/README.md

```diff
@@ -1,5 +1,11 @@
 # Allan Lee
 
+<<<<<<< HEAD
```

Comment:
@gabrielleong looks like you're editing everyone else's files :scream: see me after class and we'll try and resolve these :smiley: